spec/requests/carto/api/synchronizations_controller_spec.rb

```diff
@@ -37,7 +37,7 @@
       @headers = {
         'CONTENT_TYPE' => 'application/json'
       }
-      host! "#{@user.username}.localhost.lan"
+      host! "localhost/user/#{@user.username}"
```

Comment:
This comment is similar to the other PR. It might be that it's broken for subdomainless, I just want to check!